client/src/modules/Admin/Components/Admin.tsx

```diff
@@ -23,7 +23,18 @@ export const Admin = () => {
   const [doubleClick, setDoubleClick] = useState<boolean>(false)
 
   const [updating, setUpdating] = useState<boolean>(false);
-  type updatedStates = 'empty' | 'semester' | 'profsReset' | 'profsUpdate' | 'subjects' | 'database' | 'description';
+  type updatedStates =
```

Comment:
What are these states used for - I believe that the updating/setUpdating is used to prevent multiple database areas from being updated at once, but what do we get from knowing which states have been updated?